zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/metrics/CamundaExporterMetrics.java

```diff
@@ -35,6 +36,7 @@ public CamundaExporterMetrics(final MeterRegistry meterRegistry) {
             .publishPercentileHistogram()
             .register(meterRegistry);
     processInstancesArchived = meterRegistry.counter(meterName("archived.process.instances"));
+    batchOperationsArchived = meterRegistry.counter(meterName("archived.batch.operations"));
```

Comment:
:question: This is a new metric, right? IIRC there was no existing metric for the count of batch operations archived?